src/main/java/org/folio/service/routinglists/validators/RoutingListValidatorUtil.java

```diff
@@ -0,0 +1,34 @@
+package org.folio.service.routinglists.validators;
+
+import org.folio.rest.core.exceptions.ErrorCodes;
+import org.folio.rest.jaxrs.model.Error;
+import org.folio.rest.jaxrs.model.PoLine;
+import org.folio.rest.jaxrs.model.RoutingListCollection;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import static java.util.stream.Collectors.toList;
+
+public class RoutingListValidatorUtil {
+
+  public static List<Error> validateRoutingList(RoutingListCollection rListExisting, PoLine poLine) {
```

Comment:
thinking that better to include this code into service layer, it used only in single place, so new class is not needed